src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -308,11 +308,10 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
         this.shouldDisableOptimization();
         this.setCustomClasses();
         this.svg.addEventListener('click', this.handleItemClick);
-        this.zoomOnCurrentAsset(
-          document.querySelector(
-            `.${currentAssetClassName || (customClassNames || {}).currentAsset}`
-          )
-        );
+        const currentAssetElement = document.querySelector(
+          `.${currentAssetClassName || (customClassNames || {}).currentAsset}`
```

Comment:
@maksnester FYI `|| (customClassNames || {}).currentAsset` could also be dropped here, as `currentAssetClassName` is always defined (as you mentioned https://github.com/cognitedata/gearbox.js/pull/734/files#r764014511)

Reply:
I changed that line in the PR you mentioned 🙂